apps/token-e2e/src/integration/flow/withdrawal-flow.cy.js

```diff
@@ -0,0 +1,176 @@
+      cy.vega_wallet_import();
+      cy.deposit_asset(usdcEthAddress);
+    });
+
+    beforeEach('Navigate to withdrawal page', function () {
+      cy.visit('/');
+      cy.navigate_to('withdrawals');
+      cy.vega_wallet_connect();
+      cy.ethereum_wallet_connect();
+      waitForAssetsDisplayed(usdtName);
+    });
+
+    it('Able to open withdrawal form with vega wallet connected', function () {
+      cy.getByTestId(withdraw).should('be.visible').click();
+      cy.getByTestId(selectAsset)
+        .find('option')
+        .should('have.length.at.least', 5);
+      cy.getByTestId(ethAddressInput).should('be.visible');
+      cy.getByTestId(amountInput).should('be.visible');
+    });
+
+    it('Unable to submit withdrawal with invalid fields', function () {
+      cy.getByTestId(withdraw).should('be.visible').click();
+      cy.getByTestId(selectAsset).select('BTC (local)');
+      cy.getByTestId(balanceAvailable).should('have.text', '0.00000');
+      cy.getByTestId(submitWithdrawalButton).click();
+      cy.getByTestId(formValidationError).should('have.length', 1);
+      cy.getByTestId(useMaximum).click();
+      cy.getByTestId(submitWithdrawalButton).click();
+      cy.getByTestId(formValidationError).should(
+        'have.text',
+        'Value is below minimum'
+      );
+      cy.getByTestId(selectAsset).select(usdtName);
+      cy.getByTestId(ethAddressInput).click().type('123');
+      cy.getByTestId(amountInput).clear().click().type('10');
+      cy.getByTestId(submitWithdrawalButton).click();
+      cy.getByTestId(formValidationError).should(
+        'have.text',
+        'Invalid Ethereum address'
+      );
+    });
+
+    it('Able to withdraw asset: -eth wallet connected -withdraw funds button', function () {
+      // fill in withdrawal form
+      cy.getByTestId(withdraw).should('be.visible').click();
+      cy.getByTestId(selectAsset).select(usdtName);
+      cy.getByTestId(balanceAvailable, txTimeout).should('exist');
+      cy.getByTestId(withdrawalThreshold).should('have.text', '100,000.00000T');
+      cy.getByTestId(delayTime).should('have.text', 'None');
+      cy.getByTestId(amountInput).click().type('100');
+      cy.getByTestId(submitWithdrawalButton).click();
+
+      cy.contains('Awaiting network confirmation').should('be.visible');
+      // assert withdrawal request
+      cy.getByTestId(dialogTitle, txTimeout).should(
+        'have.text',
+        'Transaction complete'
+      );
+      cy.getByTestId(txExplorerLink)
+        .should('have.attr', 'href')
+        .and('contain', '/txs/');
+      cy.getByTestId(withdrawalAssetSymbol).should('have.text', usdcSymbol);
+      cy.getByTestId(withdrawalAmount).should('have.text', '100.00000');
+      cy.getByTestId(withdrawalRecipient)
+        .should('have.text', '0xEe7D…22d94F')
+        .and('have.attr', 'href')
+        .and('contain', '/address/0xEe7D375bcB50C26d52E1A4a472D8822A2A22d94F');
```

Comment:
some variable maybe

Reply:
I've used the existing Cypress variable for this now.